test/TaskTupleAwaiter.Tests/Adapters/AwaiterAdapter.ConfiguredTaskTupleAwaiterAdapter.cs

```diff
@@ -180,4 +180,112 @@ public override object[] GetResult()
 			return [item1, item2, item3, item4, item5, item6, item7, item8, item9, item10];
 		}
 	}
+
+	private sealed class ConfiguredTaskTupleAwaiter11Adapter(TaskTupleExtensions.TupleConfiguredTaskAwaitable<object, object, object, object, object, object, object, object, object, object, object>.Awaiter awaiter, string description) : AwaiterAdapter(description)
+	{
+		public override bool IsCompleted =>
+			awaiter.IsCompleted;
+
+		public override void OnCompleted(Action continuation) =>
+			awaiter.OnCompleted(continuation);
+
+		public override void UnsafeOnCompleted(Action continuation) =>
+			awaiter.UnsafeOnCompleted(continuation);
+
+		public override object[] GetResult()
+		{
+			var (item1, item2, item3, item4, item5, item6, item7, item8, item9, item10, item11) = awaiter.GetResult();
+			return [item1, item2, item3, item4, item5, item6, item7, item8, item9, item10, item11];
```

Comment:
@buvinghausen Aside: We could consider declaring an `object[] ITuple.ToArray()` extension method for all of these.